core/src/commonMain/kotlin/com/composeunstyled/TextField.kt

```diff
@@ -245,6 +245,7 @@ fun TextField(
     keyboardOptions: KeyboardOptions = KeyboardOptions.Default,
     visualTransformation: VisualTransformation = VisualTransformation.None,
     interactionSource: MutableInteractionSource? = null,
+    textColor: Color = Color.Unspecified,
```

Comment:
I think you would need to update your branch with the latest changes. After that, we are good to merge

Reply:
Synced my fork with your code :) Now it should be up to date